deployment/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,22 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
```

Comment:
The reason this file was added is because it is recommended to upload .lock files for versioning purposes - https://stackoverflow.com/a/67975490/6084966